test/test_utils.jl

```diff
@@ -143,6 +144,49 @@ function test_jacobians(backend; multiple_inputs=true, test_types=true)
     @test yvec == yvec2
 end
 
+function test_second_derivatives(backend; multiple_inputs=false, test_types=true)
+    if multiple_inputs
+        # ... but 
+        error("multiple_inputs=true is not supported.")
+    else
+        # explicit test that AbstractDifferentiation throws an error
+        # don't support tuple of second derivatives
+        @test_throws ArgumentError AD.second_derivative(
+            backend, x -> fder(x, yscalar), (xscalar, yscalar)
+        )
+        @test_throws MethodError AD.second_derivative(
+            backend, x -> fder(x, yscalar), xscalar, yscalar
+        )
+    end
```

Comment:
Just remove the keyword argument if it is not supported?
```suggestion
function test_second_derivatives(backend; test_types=true)
    # explicit test that AbstractDifferentiation throws an error
    # don't support tuple of second derivatives
    @test_throws ArgumentError AD.second_derivative(
        backend, x -> fder(x, yscalar), (xscalar, yscalar)
    )
    @test_throws MethodError AD.second_derivative(
        backend, x -> fder(x, yscalar), xscalar, yscalar
    )
    end
```